Update containers

AlexBaranowski opened the pull request:
## Description
Big update on the Container images.


### TLDR

- All images are built for x86_64 and arm64. ARM64 may fail at the moment; **it is not** a showstopper.
- Rocky Images now have proper README.md
- All images were migrated, so they use __our__ images
- Most of the images were updated to the newest version


### More info


- activemq: update from 6.1.4 -> 6.1.5
- angular: Update version and add TODO create init.sh in the future
- camel-k: Update from 2.3.3 to 2.5.1 -> 2.6.0 requires newer go package
- Camel-Karavan:
  ```
  [INFO] [io.quarkiverse.quinoa.deployment.packagemanager.PackageManagerRunner] npm WARN EBADENGINE Unsupported engine {
  [INFO] [io.quarkiverse.quinoa.deployment.packagemanager.PackageManagerRunner] npm WARN EBADENGINE   package: '@dagrejs/graphlib@2.2.2',
  [INFO] [io.quarkiverse.quinoa.deployment.packagemanager.PackageManagerRunner] npm WARN EBADENGINE   required: { node: '>17.0.0' },
  [INFO] [io.quarkiverse.quinoa.deployment.packagemanager.PackageManagerRunner] npm WARN EBADENGINE   current: { node: 'v16.20.2', npm: '8.19.4' }
  [INFO] [io.quarkiverse.quinoa.deployment.packagemanager.PackageManagerRunner] npm WARN EBADENGINE }
  ```
  Updated the nodejs from 16 to 22 during the build process (multi-stage build)
- Helidon -> From 3.0.6 to 4.1.6
- Jenkins -> Update from 2.479.2 to 2.492.1, fix permisson issue
- Kafka -> already the newest version
- Karaf -> Update from 4.4.6 to 4.4.7
- Kong -> Update from 3.7.1 to 3.9.0, add init.sh file
- Micronaut -> Update from 4.5.1 to 4.7.6, remove my-test.sh file
- Mongo -> update ver + fix init.sh script
- Mosquitto -> update init.sh, fix the version number containing the release
- Nodejs: updated from 20 to 22
- Openapi-generator-cli -> updated from 7.10.0 to 7.12.0
- Postgresql - 14 - still supported (about 1.5 years), update info
- Python3 - rocky 9 default 3.9 version.
- Redis - updated to 7.2.X
- Ruby - Updated to the 3.3.X
- ServiceMix -> already the newest version
- Add ARM64 build
- Better Rocky Linux README.md
- Fixes on Arm64 Build
- Enable continue-on-error for arm64
- Remove a lot of files that were not used.

## Related Issues/Tasks:

- Internal Task


## Checklist:

- [x] I have performed a self-review of my code.
- [x] I have updated the documentation (if applicable) to reflect the changes.

micnee commented on containers/images/python/README.md:
```diff
@@ -55,9 +55,8 @@ We’d love for you to contribute! You can request new features by
 creating an issue or submitting a pull request with your contribution to
```
linijka 41 -> APP_VERSION pozostało

micnee commented on containers/images/rabbitmq/README.md:
```diff
@@ -48,9 +48,8 @@ We’d love for you to contribute! You can request new features by
 creating an issue or submitting a pull request with your contribution to
```
l. 27 -> stare app version

micnee commented on containers/images/postgresql/README.md:
```diff
@@ -54,13 +54,12 @@ Please note that the ports need to be either manually forwarded with the
 
```
l. 40, l. 44 -> versiony do apdejtu

micnee commented on containers/images/openapi-generator-cli/README.md:
```diff
@@ -70,9 +70,8 @@ We’d love for you to contribute! You can request new features by
 creating an issue or submitting a pull request with your contribution to
```
l. 64 generator_version do apdejtu

micnee commented on containers/images/mosquitto/README.md:
```diff
@@ -144,9 +144,8 @@ We’d love for you to contribute! You can request new features by
 creating an issue or submitting a pull request with your contribution to
```
l. 129 -> version

micnee commented on containers/images/activemq/README.md:
```diff
@@ -49,9 +49,8 @@ We’d love for you to contribute! You can request new features by
 creating an issue or submitting a pull request with your contribution to
```
l. 29 -> version

micnee commented on containers/images/angular/README.md:
```diff
@@ -37,9 +37,9 @@ We’d love for you to contribute! You can request new features by creating an
 issue or submitting a pull request with your contribution to this image on the
```
l. 29 - image wprost nie robi expose'a portu 3000

micnee commented on containers/images/camel-karavan/README.md:
```diff
@@ -42,9 +42,8 @@ We’d love for you to contribute! You can request new features by
 creating an issue or submitting a pull request with your contribution to
```
l. 23-29: version/jre version

micnee commented on containers/images/mongodb/README.md:
```diff
@@ -44,17 +44,17 @@ We’d love for you to contribute! You can request new features by
 creating an issue or submitting a pull request with your contribution to
```
l. 29 -> version

micnee commented on containers/images/ruby/README.md:
````diff
@@ -29,13 +30,13 @@ This image uses the following environment variables:
 
 ```
 PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
-APP_VERSION="3.0.7"
````
zmiana która zapoczątkowała serię komentarzy o starszych wersjach w innych README <3

micnee commented on containers/images/helidon/README.md:
```diff
@@ -63,9 +63,8 @@ We’d love for you to contribute! You can request new features by
 creating an issue or submitting a pull request with your contribution to
```
l. 59: brak listingu exposed portów

micnee commented on containers/images/karaf/README.md:
```diff
@@ -50,9 +50,8 @@ We’d love for you to contribute! You can request new features by
 creating an issue or submitting a pull request with your contribution to
```
l. 28: version

micnee reviewed (approved):
A lot of comments have a given line in them (especially those about the readme), because it's impossible to mark unedited lines.
And apropos: I very much vote for throwing out those listings of variables/ports/other “dynamic” stuff from the readme file - there will always be something somewhere that doesn't fit and/or is somehow out of line with the Dockerfile - and anyway, probably most people interested in this will look at the Dockerfile if they want something :)

AlexBaranowski commented on containers/build-container.sh:
```diff
@@ -261,16 +261,30 @@ push_container_image(){
-        if [ "$BASE_ARCH" == "x86_64" ];then
+        # Note the x86_64 MUST BE the first build
+        if [ "$BASE_ARCH" == "x86_64" ]; then
+            echo "Removing the latest tag for multiarch"
```
Yes x86_64 build should remove any traces of older manifest files, so it can setup the multi arch.

AlexBaranowski commented:
@micnee I updated the README.md so it's obvious that version is set at the build process. Also I merged two ifs that you pointed out and pack operations in for loops.

I also added the Helidon port as requested.

THX
